### PR TITLE
Code view: give link to history and blame views of each file

### LINKED ISSUE #466: Code view: give link to history and blame views of each file
Suggestion: in the code view, we can give link to go to the history view and the blame view of a file (on GitHub)

image

e.g.,
https://github.com/reposense/RepoSense/blame/release/docs/UserGuide.md
https://github.com/reposense/RepoSense/commits/release/docs/UserGuide.md

### PR DESCRIPTION
**Pull request by fzdy1914**
Fixes #466 

```
Reviewers may have the need to investigate on a file to see who else
has edited the same file, however, this feature is not available in the
code view.

Let's add direct links in the code view to visit the history view and
blame view of the files (on GitHub).
```

### CODE REVIEW

**Comment by fzdy1914** [comment]
@reposense/devs Ready for review

File: frontend/src/static/css/style.scss [hunk 1]
```diff
@@ -292,13 +292,21 @@ header{
...
+                cursor:pointer;
+            }
+
             .loc{
                 color:mui-color('grey');
             }
+
+            .button{
+                color:mui-color("grey");
+                text-decoration:none;
```
Comment (ongspxm) [inline frontend/src/static/css/style.scss]: minor thing, but add abit of margin left here

Comment (ongspxm) [reply frontend/src/static/css/style.scss]: and maybe down the font size a little? seems a little tacky now that its pretty huge

File: frontend/src/static/js/v_authorship.js [hunk 2]
```diff
@@ -184,6 +184,26 @@ window.vAuthorship = {
...
     },
 
+    getFileBlameLink(file) {
+      const repo = window.REPOS[this.info.repo];
+
+      return `http://github.com/${
+        repo.location.organization}/${
+        repo.location.repoName}/blame/${
+        repo.branch}/${
+        file.path}`;
```
Comment (ongspxm) [inline frontend/src/static/js/v_authorship.js]: check jslint, not sure if this is the right format, dun look very pretty hahahhaahh

File: frontend/src/static/js/v_authorship.js [hunk 3]
```diff
@@ -184,6 +184,26 @@ window.vAuthorship = {
...
+
+    getFileHistoryLink(file) {
+      const repo = window.REPOS[this.info.repo];
+
+      return `http://github.com/${
+        repo.location.organization}/${
+        repo.location.repoName}/commits/${
+        repo.branch}/${
+        file.path}`;
+    },
```
Comment (ongspxm) [inline frontend/src/static/js/v_authorship.js]: i would do a combined one
```
getFileLink(file, path) {
      const repo = window.REPOS[this.info.repo];

      return `http://github.com/${
        repo.location.organization}/${
        repo.location.repoName}/${path}/${
        repo.branch}/${
        file.path}`;
    },
```

then you can call with `getFileLink(file, 'blame')` and `getFileLink(file, 'commits')`

**Comment by fzdy1914** [comment]
@damithc 
This is what the two links look like:
image

**Comment by damithc** [comment]
> @damithc
> This is what the two links look like:

Looks good to me.

**Comment by eugenepeh** [comment]
Nits to the proposed message as well:

```
Reviewers may have the need to investigate on a file to see who else
has edited the same file, however, this feature is not available in the
code view.

Let's add direct links in the code view to visit the history view and
blame view of the files (on GitHub).
```

Reminder, 
@fzdy1914 do update the image in userguide.

**Comment by yamidark** [comment]
@fzdy1914 Seems like the diagram in the DeveloperGuide [here](https://github.com/reposense/RepoSense/blob/master/docs/DeveloperGuide.md#html-dashboard)  needs to be updated as well

**Comment by fzdy1914** [comment]
> @fzdy1914 Seems like the diagram in the DeveloperGuide [here](https://github.com/reposense/RepoSense/blob/master/docs/DeveloperGuide.md#html-dashboard) needs to be updated as well

OK, will update it today.

**Pull request merged**